terraform/layer1-aws/aws-eks.tf

```diff
@@ -282,32 +221,3 @@ module "aws_ebs_csi_driver" {
 
   tags = local.tags
 }
-
-resource "aws_kms_key" "eks" {
```

Comment:
Why has kms key been deleted?

Reply:
in eks modules seted encryption for default